code/modules/flufftext/Hallucination.dm

```diff
@@ -167,10 +175,12 @@ GLOBAL_LIST_INIT(hallucination_list, list(
 		qdel(src)
 		return
 	feedback_details += "Vent Coords: [center.x],[center.y],[center.z]"
-	var/image/plasma_image = image(image_icon,center,image_state,FLY_LAYER)
+	var/obj/effect/plasma_image_holder/PIH = new(center)
```

Comment:
lowercase please

Reply:
don't we allow uppercase acronyms for temp variables?